provider/resources.go

```diff
@@ -248,9 +246,6 @@ func Provider() tfbridge.ProviderInfo {
 		Python: (func() *tfbridge.PythonInfo {
 			i := &tfbridge.PythonInfo{
 				RespectSchemaVersion: true,
-				Requires: map[string]string{
-					"pulumi": ">=3.0.0,<4.0.0",
-				},
```

Comment:
Nit: It shouldn't matter, but in some cases you're deleting the whole line like here, and in other cases you're leaving an empty map.